core-api/src/main/java/com/optimizely/ab/config/LiveVariable.java

```diff
@@ -117,7 +118,8 @@ public LiveVariable(@JsonProperty("id") String id,
         this.type = type;
     }
 
-    public @Nullable VariableStatus getStatus() {
+    public @Nullable
+    VariableStatus getStatus() {
```

Comment:
this should be single line.

the cause is probably the annotation on the return type being unconventional. Snippet below is equivalent and probably wont confuse auto-formatter tool

```java
     @Nullable
     public VariableStatus getStatus() {
```